src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/TestEnvironment.cs

```diff
@@ -56,7 +56,10 @@ static bool IsDarwin ()
 		public static bool IsRunningOnHostedAzureAgent {
 			get {
 				string agentNameValue = Environment.GetEnvironmentVariable ("AGENT_NAME");
-				return !string.IsNullOrEmpty(agentNameValue) && agentNameValue.ToUpper ().Contains ("HOSTED");
+				bool hasHostedAgentName = !string.IsNullOrEmpty (agentNameValue) && agentNameValue.ToUpper ().Contains ("AZURE PIPELINES");
```

Comment:
This should be `.ToUpperInvariant()`, to avoid the [Turkish 'I' problem](https://docs.microsoft.com/en-us/previous-versions/dotnet/articles/ms994325(v=msdn.10)#the-turkish-example) (and I suspect `.ToUpperInvariant()` was added *after* that article was written...).